lib/awful/util.lua

```diff
@@ -344,6 +344,13 @@ function util.get_rectangle_in_direction(dir, recttbl, cur)
     return grect.get_in_direction(dir, recttbl, cur)
 end
 
+-- @deprecated awful.util.pread
+function util.pread ()
+    util.deprecate("Use io.popen() directly or look at awful.spawn.easy_async() "
+            .. "for an asynchronous alternative")
+    return ""
+end
```

Comment:
I added the `---` and fixed the typo. I wasn't sure how the thing marked as a hack in `init.lua` worked. I assume it's to get a round a race condition or something, but is seems like _this_ would be the correct place for it and the other one should be dropped. Given Lua's namespacing I figured re-declaring it couldn't hurt but not having it at all might create a problem.